src/platforms/common/blackpill-f4/blackpill-f4.c

```diff
@@ -72,6 +72,13 @@ void platform_init(void)
 		scb_reset_core();
 	}
 #endif
+
+	/* Unmap ST MaskROM and map back Internal Flash */
+	rcc_periph_clock_enable(RCC_SYSCFG);
+	if ((SYSCFG_MEMRM & 3U) == 1U) {
```

Comment:
The braces are superfluous in this if statement, and the one in the bootloader, please drop them.